bookmarks.html

```diff
@@ -87,5 +87,7 @@ <h2 class="app-card__title">Question</h2>
         </li>
       </ul>
     </nav>
+    <script src="activeSite.js"></script>
+    <script language="javascript" type="text/javascript" src="activeSite.js"></script>
```

Comment:
as i looked for this in an internet search machine i read that `language="javascript"` is not used any more - i think i will put this topic (including js correctly in a project) to my question list for our head coach

Reply:
thx for checking